Makefile

```diff
@@ -504,6 +504,9 @@ clean_all: $(TARGETS_CLEAN) test_clean
 ## configs           : Hydrate configuration
 configs: configs
 
+## configs_all       : Build all configs
+configs_all: $(BASE_CONFIGS)
```

Comment:
```suggestion
## all_configs       : Build all configs
all_configs: $(BASE_CONFIGS)
```

Reply:
We have clean_all. Admittedly that’s a verb, not a noun. What do others think? I did wonder about this.

Reply:
Hence the `make all_configs` suggestion

Reply:
we already have `target-mcu`,`targets`,`targets-by-mcu`,`targets-ci-print`, so should it follow such? e.g. `targets_all` ?
sorry to add confusion :wink: 
i'd be okay with `all_configs`.

Reply:
`all_configs` is more intuitive so I've changed it.